samza-test/src/test/java/org/apache/samza/test/framework/StreamTaskIntegrationTest.java

```diff
@@ -263,4 +268,10 @@ public void genData(Map<Integer, List<KV>> inputPartitionData, Map<Integer, List
     }
   }
 
-}
\ No newline at end of file
+}
+
+class TestContext implements ExternalContext {
+  public int getMultiplier() {
+    return 10;
```

Comment:
To give a slightly better example of how `ExternalContext` might be used, maybe pass the multiplier as an argument when creating a `TestContext` instead of hardcoding it into `getMultiplier`.

Reply:
sure